recipes/intake-astro/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.1" %}
+{% set name = "intake-astro" %}
+{% set sha256 = "8f2a5f49a10a52cd58770154075de27229242e9ae5c4e95c6a449b285653b90d" %}
+
+package:
+   name: intake-accumulo
```

Comment:
I believe this is the problem. `intake-accumulo` does exists. The linter message is just reading another variable. (That is why I don't like the excess of jinja variables!)